bundler/lib/bundler/templates/newgem/.github/workflows/main.yml.tt

```diff
@@ -0,0 +1,18 @@
+name: Ruby
+
+on: [push,pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
```

Comment:
I just noticed that this is no longer the recommended action for setting up ruby, it's `ruby/setup-ruby` now.